src/modules/ordersTable/pure/OrdersTableContainer/OrderRow/index.tsx

```diff
@@ -190,7 +190,7 @@ export function OrderRow({
         </TableRowCheckboxWrapper>
       )}
       {/* Order sell/buy tokens */}
-      <styledEl.CurrencyCell clickable onClick={onClick}>
+      <styledEl.CurrencyCell clickable title={order.id} onClick={onClick}>
```

Comment:
Added a small indicator to distinguish twap orders from others in the orders table.
If the id in the title is short - it's a ComposableCoW order :) 

<img width="390" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/3afb7152-7725-47cc-936f-1dd0a9f92565">

Reply:
Is this just for debugging?

Reply:
Yes. Changed it to `data-id`, so now the tooltip won't be shown